src/components/AppSidebar/AppSidebar.vue

```diff
@@ -215,8 +215,8 @@
 						</Actions>
 					</div>
 				</div>
-				<div v-if="$slots['primary-actions'] && !empty" class="app-sidebar-header__action">
-					<slot name="primary-actions" />
+				<div v-if="$slots['description'] && !empty" class="app-sidebar-header__action">
```

Comment:
```suggestion
				<div v-if="$slots['description'] && !empty" class="app-sidebar-header__description">
```

Reply:
Ah right, we already have app-sidebar-header__desc
Maybe a better name then?

`app-sidebar-header__action` => `app-sidebar-header__description`
`app-sidebar-header__desc` => `app-sidebar-header__details`

?